admin/tools/project-analysis/app/api/readme/route.ts

```
@@ -0,0 +1,16 @@
+import { NextResponse } from 'next/server';
+import { promises as fs } from 'fs';
```

Biome (1.9.4)

[error] A Node.js builtin module should be imported with the node: protocol.

Using the node: protocol is more explicit and signals that the imported module belongs to Node.js.
Unsafe fix: Add the node: protocol.


(lint/style/useNodejsImportProtocol)